src/oci-umount.c

```diff
@@ -160,9 +159,9 @@ static int parse_mountinfo(struct mount_info **info, size_t *sz)
 	size_t len = 0;
 	int table_idx = 0;
 
-	fp = fopen(MOUNTINFO_PATH, "r");
+	fp = fopen("/proc/1/mountinfo", "r");
```

Comment:
Yeah, I just checked this is fine. Only the /proc/self isn't present.
 However, we have another issue: this logic won't work when we share pid namespaces (which is coming to cri-o very soon). 
To get around that, we could parse /proc/container_pid/mountinfo before entering the mount namespace and then join the mount namespace to umount.